content/spin/kv-store.md

```diff
@@ -85,6 +85,25 @@ $ spin new http-ts spin-key-value
 
 {{ blockEnd }}
 
+{{ startTab "Python" }}
+
+<!-- @selectiveCpy -->
+
+```bash
+# Install the py2wasm plugin
+$ spin plugin install py2wasm
+# Install the http-py template
+$ spin templates install --git https://github.com/fermyon/spin-python-sdk
+# Create the new http-py application
+$ spin new --accept-defaults http-py hello
+# Install toml, using pipenv (Note: "pip3 install pipenv" will install pipenv if necessary).
+$ pipenv install toml
```

Comment:
Why do we need `toml` here?  None of the other languages seem to add a TOML parser...?